test/files/run/settings-parse.scala

```diff
@@ -3,9 +3,8 @@ import scala.language.postfixOps
 import scala.tools.nsc._
 
 object Test {
-  val tokens        = List("", "-deprecation", "foo.scala")
-  val subsets       = tokens.toSet.subsets.toList
-  val permutations0 = subsets.flatMap(_.toList.permutations).distinct
+  val tokens        = "" :: "-deprecation" :: "foo.scala" :: Nil
+  val permutations0 = tokens.toSet.subsets.flatMap(_.toList.permutations).toList.distinct
```

Comment:
I'm already forgetting, but I do remember this was just for style points, not needing the intermediate `val subsets`. "Woo-hoo." The definition of `tokens` was changed so that no one would say, `s/List/Set`, which would be deflating.